loops/fsharp/code.fsproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
Here it is best to specify `net9.0`. It is also a good idea to specify `<PublishAot>true</PublishAot>`. When using AOT compilation, it is necessary to use `Console.WriteLine` instead of `printfn` instead.